### PR TITLE
Fixes to #274

### PR DESCRIPTION
**Pull request by RichardLitt**
@diegogsann, what do you think? I edited it heavily and moved it around a bit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -175,6 +181,79 @@ $ git remote set-url origin [url of the actual repo]
 
 For more, see [this StackOverflow topic](https://stackoverflow.com/questions/2432764/how-to-change-the-uri-url-for-a-remote-git-repository#2432799).
 
+
+### I want to add code to someone else's repository
+
+Git doesn't allow you to add code to someone else's repository without access rights. Niether does GitHub, which is not the same as Git, but rather a hosted service for Git repositories. However, you can suggest code using patches, or, on GitHub, forks and pull requests.
```
Comment (diegogsann) [inline README.md]:
```suggestion
Git doesn't allow you to add code to someone else's repository without access rights. Neither does GitHub, which is not the same as Git, but rather a hosted service for Git repositories. However, you can suggest code using patches, or, on GitHub, forks and pull requests.
```

File: README.md [hunk 2]
```diff
@@ -175,6 +181,79 @@ $ git remote set-url origin [url of the actual repo]
...
+
+If you `cd` into the resulting directory, and type `git remote`, you'll see a list of the remotes. Normally there will be one remote - `origin` - which will point to `k88hudson/git-flight-rules`. In this case, we also want a remote that will point to your fork.
+
+First, to follow a Git convention, we normally use the remote name `origin` for your own repository, and `upstream` for whatever you've forked. So, rename the `origin` remote to `upstream`
+
+```sh
+$ git remote rename origin upstream
+```
+
+You can also do this using `git remote set-url`, but it takes longer and is more strps.
```
Comment (diegogsann) [inline README.md]:
```suggestion
You can also do this using `git remote set-url`, but it takes longer and is more steps.
```

File: README.md [hunk 3]
```diff
@@ -175,6 +181,79 @@ $ git remote set-url origin [url of the actual repo]
...
+$ (feature/my-feature) git push --set-upstream origin feature/my-feature
+```
+
+There is no way to suggest a pull request using the CLI using Git (although there are tools, like [hub](http://github.com/github/hub), which will do this for you). So, if you're ready to make a pull request, go to your GitHub (or other Git host) and create a new pull request. Note that your host automatically links the original and forked repositories.
+
+After all of this, do not forget to respond to any code review feedback.
+
+#### I need to update my fork with latest updates from original repository
+
+After a while, the `upstream` repository may have been updated, and these updates need to be pulled into your `origin` repo. Remember that like you, other people are contributing too. Suppose that you are in your own feature branch and you nees to update it with the original repository updates.
```
Comment (diegogsann) [inline README.md]:
```suggestion
After a while, the `upstream` repository may have been updated, and these updates need to be pulled into your `origin` repo. Remember that like you, other people are contributing too. Suppose that you are in your own feature branch and you need to update it with the original repository updates.
```

**Comment by RichardLitt** [comment]
Thanks for the spell check! Looks great to me now. Merging.

**Pull request merged**